website_sale_b2x_alt_price/templates/product.xml

```diff
@@ -42,4 +42,36 @@
             <p class="text-muted small" t-call="website_sale_b2x_alt_price.alt_price" />
         </xpath>
     </template>
+    <template
+        id="sale_order_portal_template"
+        name="Alternative (un)taxed price"
+        inherit_id="sale.sale_order_portal_template"
+        customize_show="True"
```

Comment:
Why `customize_show`? This is a portal view, not attached to a website.

Reply:
Right, I correct it right now

Reply:
Have you pushed the change?

Reply:
Right now